R/item_deletion_h.R

```diff
@@ -0,0 +1,496 @@
+#'        \item{h CAI SFI-PFI}{A list containing two items, `ref` and `foc` 
+#'        which are matrices storing Cohen's h values quantifying the 
+#'        discrepancy between CAI under SFI vs. PFI for the reference group and
+#'        the focal group respectively, for subsets of items.}
+#'        \item{delta h SFI-PFI (deletion)}{A list containing two items, 
+#'        `ref` and `foc` which are matrices storing delta h values quantifying 
+#'        the impact of deleting an item on the discrepancy between CAI under
+#'        SFI vs. PFI for the reference group and the focal group respectively,
+#'        for subsets of items.}
+#'        \item{PartInv by groups}{Two lists (`reference` and `focal`), each 
+#'        containing restructured tables of PartInv outputs by invariance
+#'        condition, and corresponding Cohen's h values for each item deletion
+#'        scenario.}
+#'        \item{PartInv}{Two lists (`strict` and `partial`), each containing 
+#'        PartInv() outputs.}
+#'        \item{formatted}{Logical, whether simplified and formatted output  
+#'        should be printed.}
+#'        \item{return_items}{A vector containing the items that will be considered
+#'        for deletion.}
+#' @examples
+#' # Multidimensional example
+#' lambda_matrix <- matrix(0, nrow = 5, ncol = 2)
+#' lambda_matrix[1:2, 1] <- c(.322, .655)
+#' lambda_matrix[3:5, 2] <- c(.398, .745, .543)
+#' 
+#' multi_dim <- item_deletion_h(propsel = .05, n_dim = 5,
+#'                              weights_item = c(1/4, 1/4, 1/6, 1/6, 1/6),
+#'                              weights_latent = c(0.5, 0.5), 
+#'                              alpha_r = c(0, 0),
+#'                              alpha_f = c(-0.3, 0.1),
+#'                              psi_r = matrix(c(1, 0.5, 0.5, 1), nrow = 2),
+#'                              lambda_r = lambda_matrix,
+#'                              nu_r = c(.225, .025, .010, .240, .125),
+#'                              nu_f = c(.225, -.05, .240, -.025, .125),
+#'                              Theta_r = diag(1, 5),
+#'                              Theta_f = diag(c(1, .95, .80, .75, 1)),
+#'                              plot_contour = TRUE,
+#'                              print_formatted = TRUE)
+#' # Single dimension example
+#' single_dim <- item_deletion_h(propsel = .10,
+#'                                weights_item = c(1, 0.9, 0.8, 1),
+#'                                weights_latent = 0.9,
+#'                                alpha_r = 0.5,
+#'                                alpha_f = 0,
+#'                                psi_r = 1,
+#'                                lambda_r = c(.3, .5, .9, .7),
+#'                                nu_r = c(.225, .025, .010, .240),
+#'                                nu_f = c(.225, -.05, .240, -.025),
+#'                                Theta_r = diag(.96, 4),
+#'                                n_dim = 1, plot_contour = TRUE,
+#'                                print_formatted = TRUE)
```

Comment:
Now that we use the `itemdeletion` class with a print method, I'm curious whether you think we should keep the `print_formatted` option? There are at least three options:

1. Keep it for both `item_deletion_h()` and `print.itemdeletion()`
2. Keep it only for `print.itemdeletion()`
3. Not keeping it anywhere.